server/routes/index.mjs

```diff
@@ -48,12 +56,44 @@ const getDomains = asyncMiddleware.wrap(async (req, res) => {
   }
 
   const domains = await Promise.all(companies.map(async (company) => {
-    const result = await searchCompany(company);
-    const { domain, tld } = parseDomain(result.link);
-    return {
+    const foundCompanies = await searchCompanies(company);
+    const isExcludedCompany = excludedWebsites.find(excludedWebsite => excludedWebsite.toUpperCase().includes(company.toUpperCase()));
```

Comment:
`excludedWebsite.toUpperCase()` - you can do this once when you initialize the array.

Reply:
The return value is not a boolean as the name of the variable applies. Consider changing the variable name.